cmd/snap-confine/mount-support.c

```diff
@@ -878,6 +903,41 @@ static bool __attribute__((used))
 	return false;
 }
 
+static struct sc_mount *sc_init_homedir_mounts(const struct sc_invocation *inv)
```

Comment:
sorry, I somehow thought this was setting things but it actually returns them, so it can be called just sc_homedir_mounts

Reply:
That makes sense, thank you for catching that